recipes/onetbb/all/conanfile.py

```diff
@@ -120,7 +120,7 @@ def generate(self):
         toolchain.variables["TBB_DISABLE_HWLOC_AUTOMATIC_SEARCH"] = not self._tbbbind_build
         if self._tbbbind_explicit_hwloc:
             hwloc_package_folder = self.dependencies["hwloc"].package_folder
-            hwloc_lib_name = "hwloc.lib" if self.settings.os == "Windows" else "libhwloc.so"
+            hwloc_lib_name = "hwloc.lib" if self.settings.os == "Windows" else "libhwloc.dylib" if self.settings.os == "Macos" else "libhwloc.so"
```

Comment:
I would split it into a multi-line statement.

Reply:
done